core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3085,7 +3085,8 @@ private[spark] object Utils
         entry = in.getNextEntry()
       }
       in.close() // so that any error in closing does not get ignored
-      logInfo(log"Unzipped from ${MDC(PATH, dfsZipFile)}\n\t${MDC(PATHS, files.mkString("\n\t"))}")
+      logInfo(log"Unzipped from ${MDC(PATH, dfsZipFile)}")
```

Comment:
Lets just move this line to `DEBUG`. We can't corelate easily if we split the lines

Reply:
Done